src/AppInstallerCLIE2ETests/UpgradeCommand.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace AppInstallerCLIE2ETests
+{
+    using NUnit.Framework;
+    using System.IO;
+
+    public class UpgradeCommand : BaseCommand
+    {
+        [Test]
+        public void UpgradePortable()

```

Comment:
Should there also be tests for when the ARP data is mismatched?
* PortableUpgradeARPMismatch
* PortableUpgradeForcedOverride

Reply:
Added suggested tests